Repositories/EmbedRepository.cs

```diff
@@ -60,31 +61,81 @@ public class EmbedRepository : Repository<EmbeddedProject>, IEmbedRepository
         public EmbedRepository(DbContext dbContext) : base(dbContext) { }
 
         /// <summary>
-        /// Gets the embedded project asynchronous.
+        /// Redacts the user.
+        /// </summary>
+        /// <param name="embeddedProject">The embedded project.</param>
+        /// <returns></returns>
+        private EmbeddedProject RedactUser(EmbeddedProject embeddedProject)
+        {
+            if(embeddedProject?.Project?.User?.IsPublic == false)
+            {
+                embeddedProject.Project.User.Email = Defaults.Privacy.RedactedEmail;
+            }
+            if(embeddedProject?.User?.IsPublic == false)
+            {
+                embeddedProject.User.Email = Defaults.Privacy.RedactedEmail;
+            }
+
+            return embeddedProject;
+        }
+
+        /// <summary>
+        /// Redacts the user.
+        /// </summary>
+        /// <param name="embeddedProjects">The embedded projects.</param>
+        /// <returns></returns>
+        private List<EmbeddedProject> RedactUser(List<EmbeddedProject> embeddedProjects)
+        {
+            for(int i = 0; i < embeddedProjects.Count; i++)
+            {
+                embeddedProjects[i] = RedactUser(embeddedProjects[i]);
+            }
+            return embeddedProjects;
+        }
+
+        /// <summary>
+        /// Finds the asynchronous.
+        /// </summary>
+        /// <param name="id">The identifier.</param>
```

Comment:
The identifier of what?

The documentation in this class (mostly the parameter documentation) is very vague. Please add a proper description of what the parameters do.